src/guides/v2.2/ui_comp_guide/components/ui-hidden.md

```diff
@@ -12,3 +12,31 @@ The Hidden component is a form element that implements the [HTML](https://glossa
 | `component` | The path to the component’s JS constructor in terms of RequireJS. | String | `Magento_Ui/js/form/element/abstract` |
 | `template` | The path to the component’s `.html` template. | String | `ui/form/element/hidden` |
 
+## Source files
+
+Extends [`UiElement`]({{ page.baseurl }}/ui_comp_guide/concepts/ui_comp_uielement_concept.html):
+
+-  [app/code/Magento/Ui/view/base/web/js/form/element/abstract.js]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Ui/view/base/web/js/form/element/abstract.js)
+-  [app/code/Magento/Ui/view/base/web/templates/form/element/hidden.html]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Ui/view/base/web/templates/form/element/hidden.html)
+
+### Integrate the Hidden component with the Form component
```

Comment:
Hi @serhiyzhovnir. Let's add another h2 header "## Examples" and put the "### Integrate the Hidden component with the Form component" there.

So we have more or less the same structure as everywhere (ex. https://devdocs.magento.com/guides/v2.2/ui_comp_guide/components/ui-search.html)

Thank you!